models/models.js

```diff
@@ -24,6 +24,21 @@ exports.selectArticleByID = async (article_id) => {
   const {rows} = await db.query(queryStr, [article_id]);
   const article = rows[0];
   return article;
+};
 
-}
+exports.updateArticleByID = async (inc_votes, article_id) => {
+  const queryStr = format(`
+  UPDATE articles
+  SET votes = votes + $1 
+  WHERE article_id = $2
+  RETURNING *;`);
```

Comment:
`format` here isn't doing anything as you're not passing in any data for `pg-format` to use in the query!

as you are only replacing values (`$1` and `$2` replacing numbers) you don't need `pg-format` at all here!

You can just put this string in `db.query`